README.md

```diff
@@ -69,13 +69,13 @@ Packaging for your favorite distribution would be a welcome contribution!
 
 ### On Linux:
 
-(Tested on Ubuntu 16.04 x86, 16.10 x64, Gentoo x64 and Linux Mint 18 "Sarah" - Cinnamon x64)
+(Tested on Ubuntu 16.04 x86, 17.10 x86, 16.10 x64, Gentoo x64 and Linux Mint 18 "Sarah" - Cinnamon x64)
 
 1. Install Monero dependencies
 
   - For Ubuntu and Mint
 
-	`sudo apt install build-essential cmake libboost-all-dev miniupnpc libunbound-dev graphviz doxygen libunwind8-dev pkg-config libssl-dev libzmq3-dev`
+	`sudo apt install build-essential cmake libboost-all-dev miniupnpc libunbound-dev graphviz doxygen libunwind8-dev pkg-config libssl-dev libzmq3-dev libreadline-dev`
```

Comment:
readline was removed from dependencies in #1174.

Reply:
There seems to be some problem though, as I was unable to build without it (on latest branch, after the referenced commit). Maybe it is not a dependency of monero/monero-gui, but it still is a dependency for the build process?

Reply:
The GUI definitely doesn't need readline since it's not a CLI app. Maybe we should add `-DUSE_READLINE=OFF` to the arguments of CMake in `get_libwallet_api.sh`?